sources/packages/web/src/views/aest/SearchInstitutions.vue

```diff
@@ -1,82 +1,94 @@
-          />
-        </v-col>
-        <v-col
-          ><v-btn
-            :disabled="!legalName && !operatingName"
-            color="primary"
-            data-cy="searchInstitutions"
-            @click="searchInstitutions"
-            >Search</v-btn
-          ></v-col
-        >
-      </v-row>
-    </content-group>
-
-    <content-group v-if="institutionsFound" class="mt-8">
-      <DataTable
-        v-if="institutionsFound"
-        class="mt-4"
-        :autoLayout="true"
-        :value="institutions"
-      >
-        <Column field="legalName" header="Legal Name" :sortable="true">
-          <template #body="slotProps">
-            <div class="p-text-capitalize">
-              {{ slotProps.data.legalName }}
-            </div>
-          </template>
-        </Column>
-        <Column field="operatingName" header="Operating Name" :sortable="true">
-          <template #body="slotProps">
-            <div class="p-text-capitalize">
-              {{ slotProps.data.operatingName }}
-            </div>
-          </template>
-        </Column>
-        <Column field="address" header="Address">
-          <template #body="slotProps">
-            <div class="p-text-capitalize">
-              {{ getFormattedAddress(slotProps.data.address) }}
-            </div>
-          </template>
-        </Column>
-        <Column>
-          <template #body="slotProps">
-            <v-btn
+    <body-header
+      title="Search Institution"
+      subTitle="Look up an institution by entering their information below."
+    >
+    </body-header>
+    <v-form ref="searchInstitutionsForm">
+      <content-group class="mb-8">
+        <v-row>
+          <v-col>
+            <v-text-field
+              density="compact"
+              label="Legal name"
+              variant="outlined"
+              v-model="legalName"
+              data-cy="legalName"
+              @keyup.enter="searchInstitutions"
+              hide-details
+            />
+          </v-col>
+          <v-col>
+            <v-text-field
+              density="compact"
+              label="Operating name"
+              variant="outlined"
+              v-model="operatingName"
+              data-cy="operatingName"
+              @keyup.enter="searchInstitutions"
+              hide-details
+            />
+          </v-col>
+          <v-col
+            ><v-btn
               color="primary"
-              data-cy="viewInstitution"
-              @click="goToViewInstitution(slotProps.data.id)"
-              >View</v-btn
-            >
-          </template>
-        </Column>
-      </DataTable>
-    </content-group>
+              data-cy="searchInstitutions"
+              @click="searchInstitutions"
+              >Search</v-btn
+            ></v-col
+          >
+        </v-row>
+        <v-input :rules="[isValidSearch()]" hide-details="auto" error />
+      </content-group>
+    </v-form>
+    <template v-if="institutionsFound">
```

Comment:
I didn't  get it. this template tag is an extension for which component?

Reply:
https://vuejs.org/guide/essentials/conditional.html#v-if-on-template